From a777b5332b184b5cb472514d3507230e9bebf5c0 Mon Sep 17 00:00:00 2001
From: Vincent Schwalbach <mail@vincent-schwalbach.de>
Date: Tue, 16 Jul 2024 13:42:59 +0200
Subject: [PATCH] feat: add HPE Nimble multipath configuration Added HPE Nimble
 Configuration from the HPE Storage Toolkit for Linux

Signed-off-by: Vincent Schwalbach <mail@vincent-schwalbach.de>
(cherry picked from commit be6922f1626f2a3583872a4c51fef6dff07cdea3)
---
 multipath/multipath.conf | 17 ++++++++++++++++-
 1 file changed, 16 insertions(+), 1 deletion(-)

diff --git a/multipath/multipath.conf b/multipath/multipath.conf
index 00a8f488..5800e362 100644
--- a/multipath/multipath.conf
+++ b/multipath/multipath.conf
@@ -156,4 +156,19 @@ devices {
         prio                        alua
         failback                    immediate
     }
-}
+    device {
+        vendor                      "Nimble"
+        product                     "Server"
+        path_grouping_policy        group_by_prio
+        prio                        "alua"
+        hardware_handler            "1 alua"
+        path_selector               "service-time 0"
+        path_checker                tur
+        no_path_retry               30
+        failback                    immediate
+        fast_io_fail_tmo            5
+        dev_loss_tmo                infinity
+        rr_min_io_rq                1
+        rr_weight                   uniform
+    }
+}
\ No newline at end of file
